.devcontainer/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright (c) 2023, Arm Limited and Contributors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HERE="$(dirname "$0")"
+BUILD_VERSION="0.6.35"
```

Comment:
Can we add the build version to the script options too?

Reply:
Done